engine/table/src/main/java/io/deephaven/engine/table/impl/sources/UnionSourceManager.java

```diff
@@ -427,13 +434,23 @@ else if (nextCurrentKey == nextModifiedKey) {
                 }
                 // Existing constituent processing
                 else {
-                    processExisting(nextCurrentValue);
+                    try {
+                        processExisting(nextCurrentValue);
+                    } catch (ConstituentTableErrorException ex) {
+                        constituentExceptions = collectConstituentException(constituentExceptions, ex);
+                    }
                     advanceCurrent();
                     ++nextCurrentSlot;
                     ++nextPreviousSlot;
                 }
             }
 
+            if(constituentExceptions != null) {
+                throw new UncheckedDeephavenException(
+                        new MultiException("Errors occurred processing constituent tables:",
```

Comment:
io.deephaven.util.MultiException#maybeWrapInMultiException(java.lang.String, java.lang.Throwable...) so we don't wrap a multiexception for one thing